src/porepy/fracs/simplex.py

```diff
@@ -618,13 +625,19 @@ def __merge_domain_fracs_2d(dom, frac_p, frac_l):
     # Also add a tag to the fractures, signifying that these are fractures
     frac_l = np.vstack((frac_l, const.FRACTURE_TAG * np.ones(frac_l.shape[1])))
 
+    # Also add a tag to the subdomains, signifying that these are subdomains
+    subdom_l = np.vstack((subdom_l, const.AUXILIARY_TAG * np.ones(subdom_l.shape[1])))
+
     # Merge the point arrays, compartment points first
-    p = np.hstack((dom_p, frac_p))
+    p = np.hstack((dom_p, frac_p, subdom_p))
 
     # Adjust index of fracture points to account for the compartment points
     frac_l[:2] += dom_p.shape[1]
 
-    l = np.hstack((dom_l, frac_l)).astype("int")
+    # Adjust index of subdomains points to account for the compartment points
```

Comment:
Terminology: Compartment is old wording, we should probably use subdomains instead.

Reply:
oki